shfmt.sh

```diff
@@ -26,8 +31,12 @@ while [[ $# -gt 0 ]] && [[ ."$1" = .-*  || ."$1" = .--* ]]; do
     esac
 done
 
+command -v file > /dev/null
+command -v grep > /dev/null
```

Comment:
I believe it will be better to change the loop to do:
```
for file in "$@"; do
	filetype=$(file "$file" | grep -iq 'shell script')
	if $filetype; then
		shfmt -l -w $indentation "$file"
	fi
done
```
That way, we dont have to do any extra checks, it just fails as expected if this command is missing.